content/kapacitor/v1.5/administration/security.md

```diff
@@ -213,6 +214,9 @@ Similar commands:
 
 `curl -kv -d '{ "set": { "subscription-protocol": "https" } }' https://localhost:9092/kapacitor/v1/config/influxdb/`
 
+* If Kapacitor is behind a reverse proxy, set the `subscription-path` to append to the InfluxDB subscription URL:
+curl -kv -d '{ "set": { "subscription-path": "/path/behind/reverse-proxy" } }' https://localhost:9092/kapacitor/v1/config/influxdb/
```

Comment:
This needs to be formatted as code. All these commands would look better in codeblocks. As inline code, they will wrap on narrower screens.